src/test/scala/org/kibanaLoadTest/helpers/ESClient.java

```diff
@@ -70,21 +72,43 @@ public void createIndex(String indexName, Json source) {
         CreateIndexRequest req = CreateIndexRequest.of(b -> b.index(indexName).withJson(new StringReader(source.toString())));
         try {
             boolean created = client.indices().create(req).acknowledged();
-            logger.error(String.format("Index '%s' created %b", indexName, created));
+            if (created) {
+                logger.info(String.format("Created index '%s'", indexName));
+            } else {
+                logger.error(String.format("Failed to create index '%s'", indexName));
+            }
+        } catch (IOException e) {
```

Comment:
I think we could improve the error message and avoid duplicate code with multi-catch (available since Java 7):


```java
try {
    // ...
} catch (IOException | ElasticsearchException ex) {
    throw new RuntimeException(String.format("Failed to create index '%s'", indexName), e);
}